python/desc/sims/GCRCatSimInterface/DatabaseEmulator.py

```diff
@@ -198,7 +203,29 @@ def _transform_catalog(self, gc):
         gc.add_quantity_modifier('sindex::disk', gc.get_quantity_modifier('sersic_disk'))
         gc.add_quantity_modifier('sindex::bulge', gc.get_quantity_modifier('sersic_bulge'))
 
-        return None
+        additional_postfix = ()
```

Comment:
Please add a few lines to the `_transform_catalog` docstring explaining what `additional_postfix` is.

Reply:
done